application/controllers/SurveyAdministrationController.php

```diff
@@ -2183,7 +2184,9 @@ public function actionCopy()
                     );
                 }
             } elseif ($action == 'copysurvey' && !$aData['bFailed']) {
-                $aImportResults = XMLImportSurvey('', $copysurveydata, $sNewSurveyName, sanitize_int(App()->request->getParam('copysurveyid')), (Yii::app()->request->getPost('copysurveytranslinksfields') == '1'));
+                $copyResources = Yii::app()->request->getPost('copysurveytranslinksfields') == '1';
```

Comment:
`$copyResources = Yii::app()->request->getPost('copysurveytranslinksfields'); `

or 

`$copyResources = boolval(Yii::app()->request->getPost('copysurveytranslinksfields')); `

Reply:
Thos could be a very subjective discussion... but why any of those options is better?

Reply:
== 1 is clearly uneeded.
and i think scrutinizer send a warning about usage of == and not === 

We check a boolean

Reply:
I prefer to leave it like that. It was like these before. There are many other places on the current file where this is done like that. Prefer to leave it like that for consistency (by the way, I think is a bit safer as it is a tighter check)